### ISSUE TITLE
My humble opinion.

### ISSUE THREAD

**Issue by ImperaZim** [SharpnessXI/ElsonHomes#1]
There are some things that bother me a little about the plugin, such as: I don't understand exactly why HomeInstance::isOwner has the optional PluginBase type and the lack of some more current PHP implementations to optimize... But you did a great job considering that it is your first public plugin.

**Comment by SharpnessXI** [comment]
yeah thanks I havent learnt php fully and seen pocketmines library so thats my bad

**Comment by octoberssss** [comment]
Bros gpt subscription expired
